bubble_sort/Cpp/bubble_sort.cpp

```diff
@@ -1,4 +1,5 @@
-#include <stdio.h>
+#include <iostream>
+using namespace std;
 
 void swap(int *xp, int *yp) {
 	int temp = *xp;
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.cpp`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpxn6wlvp6/bubble_sort/Cpp/bubble_sort.cpp
+++ b/tmp/tmpxn6wlvp6/bubble_sort/Cpp/bubble_sort.cpp
@@ -17,7 +17,7 @@
 				swap(&arr[j], &arr[j+1]);
 				swapped = true;
 			}
-		}		
+		}
 		if (swapped == false)
 			break;
 	}
```